_PRESS-RELEASE.md

```diff
@@ -32,13 +32,15 @@ Once the project moves into development, the press release can be used as a touc
   > Describe how your product elegantly solves the problem.
```

Comment:
You are missing the rest of the file above the quote section. Rebase on your fork.